tabbycat/notifications/migrations/0007_migrate_data.py

```diff
@@ -2,32 +2,33 @@
 # Extended manually to create BulkNotification objects and Message-IDs
 import email
 import itertools
-import re
+import operator
 
 from django.db import migrations
 
 
 def create_bulk_notifications(apps, schema_editor):
-    SentMessageRecord = apps.get_model("notifications", "SentMessageRecord")
-    BulkNotification = apps.get_model("notifications", "BulkNotification")
+    SentMessageRecord = apps.get_model("notifications", "SentMessageRecord")  # noqa: N806
```

Comment:
Aren't the migrations excluded from flake8?

Reply:
umm... yeah. I think it might be my local configuration I need to fix.

Reply:
They're only really excluded for the long lines in the automatic migrations though; in principle I'd like data migrations to be compliant.